cloudpickle/compat.py

```diff
@@ -7,7 +7,7 @@
         from pickle5 import Pickler  # noqa: F401
     except ImportError:
         import pickle  # noqa: F401
-        from pickle import _Pickler as Pickler  # noqa: F401
+        from pickle import Pickler as Pickler  # noqa: F401
```

Comment:
```suggestion
        from pickle import Pickler  # noqa: F401
```

Reply:
I don't understand why the tests pass on Python 3.6 and 3.7 with this change: `pickle._Pickler` is the slow pure-Python implementation and I thought it was necessary for cloudpickle to customize the pickling of locally defined functions for those Python versions.

Any idea what's going on @pierreglaser?

Reply:
I've reverted this change. I missed the subtle difference between `from pickle import _Pickler` and `from _pickle import Pickler`.
I've only tested the latter to be identical to `from pickle import Pickler` on  default python installs